src/Microsoft.AspNetCore.Mvc.Razor.Extensions/ExtensionInitializer.cs

```diff
@@ -10,6 +11,9 @@ internal class ExtensionInitializer : RazorExtensionInitializer
         public override void Initialize(RazorProjectEngineBuilder builder)
         {
             RazorExtensions.Register(builder);
+
+            // Intentionally registering it after RazorExtensions. This should weed out ordering related issues when we expected components to win
+            ComponentExtensions.Register(builder);
```

Comment:
I don't get this. Components are supposed to be a built in part of the compiler.

Reply:
Moved to `RazorProjectEngine`. It's registered by default now.